demo/src/screens/componentScreens/TabControllerScreen/tab3.tsx

```diff
@@ -13,7 +14,16 @@ class Tab2 extends Component {
       this.setState({loading: false});
     }, 1200);
 
-    // this.slow();
+    /* Uncomment to test TabPage freeze functionality when the page lose focus */
+    // setInterval(() => {
+    //   this.setState({counter: this.state.counter + 1});
+    // }, 1000);
+  }
+
+  componentDidUpdate() {
+    if (this.state.counter % 3 === 0) {
+      console.warn('freeze counter', this.state.counter);
+    }
```

Comment:
Not sure about the logic here, but either way: did you leave it uncommented on purpose?

Reply:
Yes, but I can comment it as well

Reply:
done